cmd/query/app/static_handler.go

```diff
@@ -149,16 +149,17 @@ func (sH *StaticAssetsHandler) watch() {
 					}
```

Comment:
could we please refactor this nested `go func()` into a top function? There are 5 levels of indentation/nesting here.

Reply:
Sure.

Reply:
Done, I separated the `go func()` into another function. That removes some indentation levels.